modules/components/src/AdvancedSqonBuilder/filterComponents/BooleanFilter.js

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Component from 'react-component-component';
+
+import {
+  getOperationAtPath,
+} from '../utils';
+import BooleanAgg from '../../Aggs/BooleanAgg';
+import { FilterContainer } from './common';
+import './FilterContainerStyle.css';
+
+const getFieldDisplayName = (fieldDisplayNameMap, initialFieldSqon) => {
+  return fieldDisplayNameMap[initialFieldSqon.content.field] || initialFieldSqon.content.field;
+};
+
+const AggsWrapper = ({ children }) => (
+  <div className="aggregation-card">{children}</div>
+);
+
+export const BooleanFilterUI = props => {
+  const {
+    onSubmit = sqon => {},
+    onCancel = () => {},
+    ContainerComponent = FilterContainer,
+    sqonPath = [],
+    initialSqon = null,
+    field = '',
+    fieldDisplayNameMap = {},
+  } = props;
+
+  const initialState = {
+    selectedValue: undefined,
+    localSqon: initialSqon
+  };
+
+  const initialFieldSqon = getOperationAtPath(sqonPath)(initialSqon) || {
+    op: 'and',
+    content: { value: initialState.selectedValue, field },
+  };
+
+  const onSqonSubmit = s => () => onSubmit(s.state.localSqon);
+
+  const onSelectionChange = s => ({ value, generateNextSQON }) => {
+    s.setState({
+      selectedValue: value.key_as_string,
+      localSqon: generateNextSQON(s.state.localSqon),
+    });
+  };
+
+  const isActive = s => ({ value }) => value === s.state.selectedValue;
+
+  return (
+    <Component initialState={initialState}>
+      {s => (
+        <ContainerComponent onSubmit={onSqonSubmit(s)} onCancel={onCancel}>
+          <React.Fragment>
+            <div key="header" className="contentSection">
+              <span>
+                { `Participant is a ${getFieldDisplayName(fieldDisplayNameMap, initialFieldSqon)}`}
+              </span>
+            </div>
+            <div key="body" className="contentSection meowContainer">
```

Comment:
Yeah, anything named `meow`, `blah` or `potato` might be an accidentally committed temporary value. Naming stuff is hard sometime. :laughing: 

:cat: Fixed that since.